src/js/structs/Service.js

```diff
@@ -126,6 +126,15 @@ module.exports = class Service extends Item {
   getServiceStatus() {
     let {tasksRunning} = this.getTasksSummary();
     let deployments = this.getDeployments();
+    let queue = this.getQueue();
+
+    if (queue != null && queue.delay && queue.delay.overdue) {
```

Comment:
No documentation please for such things (that nobody reads).

To make it more descriptive we could refactor the code, which makes it even faster:
```JavaScript
if (queue != null && queue.delay) {
  if (queue.delay.overdue) {
     return ServiceStatus.WAITING;
  }
  return ServiceStatus.DELAYED;
}
```